### DISCUSSION TITLE
.Net 7 Performance Improvements Blog Post

### DISCUSSION THREAD

**Discussion by sirinath** [dotnet/runtime#73903] [General] [answered]
Previous years there was a post on .net performance improvements. I have been waiting for the .net 7 edition. When will this be posted?

**Comment by danmoseley** [comment, marked as answer]
@stephentoub May be planning one.

**Reply by stephentoub** [reply]
I'm in the process of writing it.

**Comment by stephentoub** [comment]
Here it is:
https://devblogs.microsoft.com/dotnet/performance_improvements_in_net_7/
